65black/StackOlite
# #160199315 Allow Users Signup

#### What does this PR do?
This PR adds functionality to the signup page.

#### Description of Task to be completed
Allows the user to signup to StackOverflow-lite.

#### How should this be manually tested?
Go to the [signup page](https://65black.github.io/StackOlite/index.html) and try to register a new account.

- Fill in the required fields (All of them) and click on signup.
- You should get redirected to the homepage.

#### What are the relevant pivotal tracker stories?
#160199315
